.github/workflows/csharp_client_compatibility.yaml

```diff
@@ -79,11 +80,13 @@ jobs:
         working-directory: client
         env:
           HAZELCAST_SERVER_VERSION: ${{ matrix.version }}
-      - name: Run all tests
+      - name: Run all tests        
         shell: pwsh
         if: ${{ matrix.kind == 'enterprise' }}
-        run: ./hz.ps1 -enterprise -server ${{ matrix.version }} test ${{ secrets.GH_PAT }}
-        working-directory: client
+        run: ./hz.ps1 -enterprise -noRestore -localRestore -server ${{ matrix.version }} test ${{ secrets.GH_PAT }}
```

Comment:
what are these?

Reply:
The secret is to download certificates from the private repo. `hz.ps1` can download the certificates by its own.